discord/message.py

```diff
@@ -1768,6 +1971,10 @@ class Message(PartialMessage, Hashable):
         The poll attached to this message.
 
         .. versionadded:: 2.4
+    message_snapshots: Optional[List[:class:`MessageSnapshot`]]
```

Comment:
```suggestion
    message_snapshots: List[:class:`MessageSnapshot`]
```

`Optional[List[X]]` is silly.

Reply:
Updated on the latest commit.